src/utils/config.ts

```diff
@@ -99,11 +100,15 @@ const setConfig = ({key, value}: ConfigUpdate): Nullable<string> => {
         case CONFIG_KEY.INITIAL_TAB_NAME:
             window.localStorage.setItem(CONFIG_KEY.INITIAL_TAB_NAME, value.toString());
             break;
-        case CONFIG_KEY.THEME:
-            throw new Error(`"${key}" cannot be managed using these utilities.`);
         case CONFIG_KEY.PAGE_SIZE:
             window.localStorage.setItem(LOCAL_STORAGE_KEY.PAGE_SIZE, value.toString());
             break;
+        /* c8 ignore start */
+        case CONFIG_KEY.THEME:
+            // Unexpected execution path.
```

Comment:
Discussed at https://github.com/y-scope/yscope-log-viewer/pull/135/files#r1876734225 : we allocate the types and enums for theme config only to avoid name conflicts and we expected only JoyUI to read / write into localStorage, which might subject to change in the future.